src/code/audio_heap.c

```diff
@@ -1204,11 +1204,11 @@ void AudioHeap_DiscardSampleCacheEntry(SampleCacheEntry* entry) {
     }
 }
 
-void AudioHeap_UnapplySampleCache(SampleCacheEntry* entry, SoundFontSample* sample) {
-    if (sample != NULL) {
-        if (sample->sampleAddr == entry->allocatedAddr) {
-            sample->sampleAddr = entry->sampleAddr;
-            sample->medium = entry->origMedium;
+void AudioHeap_UnapplySampleCache(SampleCacheEntry* entry, SoundFontSampleHeader* sampleHeader) {
+    if (sampleHeader != NULL) {
```

Comment:
does it make sense to change `sample` to `sampleHeader` in the context of audio?

genuine question cause I dont know much of how this works. But to look at a parrallel example on the gameplay side of things, you have `AnimationHeader` as a structure. However, in all cases, it is more helpful to think of it as the animation itself, because thats how the code will interface with them. So you would have a function that for example takes `AnimationHeader* anim`.

If that parallel is similar, then maybe it would be more clear to say `SoundFontSampleHeader* sample`

Reply:
The motivation for switching `sample` -> `sampleHeader` was because there are a lot of variables with `sample` in it, like `sampleData`, `sampleBank`, `sampleCache`, `sampleDma`, `sampleAddr`,

It seemed like meta data to store a collection of sample-related variables for the sample stored in the AudioTable on the cartridge.

If header info like that tends to not have "header" in the variable name and just in the struct name, then yeah I can switch to `SoundFontSampleHeader* sample` if it's clearer/consistent

Reply:
I guess the question is really, does it matter more in these contexts that its a header, or is it the high level way that the code will refer to a sample, in general. I cant make that call with the audio knowledge that I have

Reply:
Maybe others who also know audio decently well can comment, but I personally always wondered if `sample` was a pointer to the sample itself or I always had to look up the struct to see what `sample` contained. `sampleHeader` seemed clearer in my mind to give a sense on how it relates to samples in general, but yeah open to others opinions